lite/helpers.go

```diff
@@ -78,6 +78,9 @@ func (v ValKeys) signHeader(header *types.Header, first, last int) *types.Commit
 
 	// fill in the votes we want
 	for i := first; i < last; i++ {
```

Comment:
let's add `&& i < len(v)` instead of if condition

Reply:
SGTM, thanks!